sjb/config/test_cases/azure_build_node_image_centos_311.yml

```diff
@@ -0,0 +1,112 @@
+---
```

Comment:
This shouldn't be `*centos_311.yml` but `*centos.yml` like all other master jobs.